spec/mongoid/persistable/savable_spec.rb

```diff
@@ -270,31 +270,13 @@
           truck.crates.first.toys.build(name: "Teddy bear")
           truck.crates.build(volume: 0.8)
 
-          # The following is equivalent to the two lines above:
-          #
-          # truck.crates_attributes = {
-          #   '0' => {
-          #     "toys_attributes" => {
-          #       "0" => {
-          #         "name" => "Teddy bear"
-          #       }
-          #     },
-          #     "id" => crate.id.to_s
-          #   },
-          #   "1" => {
-          #     "volume" => 0.8
-          #   }
-          # }
-
```

Comment:
Why is this removed?

Reply:
It's just a dead-weight comment. Why do we need it?